themes/openy_themes/openy_rose/templates/node/node--alert--node-alert-footer.html.twig

```diff
@@ -95,7 +95,9 @@
         </div>
       </div>
       <div class="col-xs-12 col-sm-5 col-md-4 col-lg-3 site-alert__cta">
-        {{ content.field_alert_link }}
+        <div class="field-alert-link" style="">
+          <a href="{{ content.field_alert_link.0['#url'] }}" style="color: {{ text_color }}; border-color: {{ text_color}}">{{ content.field_alert_link.0['#title'] }}</a>
```

Comment:
Can we set those variable at the top of template? If we even see `content.field_alert_link.0['#url']` or something like this, this should be declared as a variable at the beginning of template.